react/package.json

```diff
@@ -3,13 +3,20 @@
   "version": "0.0.0",
   "private": true,
   "license": "UNLICENSED",
+  "scripts": {
+    "test": "vtex-test-tools"
+  },
   "dependencies": {
     "react": "^16.13.0",
     "react-apollo": "^3.1.3"
   },
   "devDependencies": {
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
-    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.93.2/public/@types/vtex.render-runtime"
+    "@vtex/test-tools": "^3.0.3",
+    "vtex.checkout-graphql": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-graphql@0.24.0+build1583340197/public/@types/vtex.checkout-graphql",
```

Comment:
I think you might need to unlink `checkout-graphql` and `checkout-resources` before running `vtex setup`.

Reply:
I will!

Reply:
done :-)